index.html

```diff
@@ -76,14 +76,14 @@ <h3 class="socialMedia">LET’S CONNECT</h3>
     <!-- ####### Works Section #######-->
     <section class="works">
```

Comment:
- [ ] You have done a great job with your portfolio accessibility but there is an issue with the contact form button contrast. When it's in a hover state it can be lost with the background color. I would suggest updating the hover color for this element for a better accessibility :raised_hands:

  Your button:
  ![image](https://user-images.githubusercontent.com/81522435/139907724-0c19c300-1295-4ece-98cd-17d640004dae.png)

Reply:
Done.